src/components/strategies/edit/EditPriceNav.tsx

```diff
@@ -1,64 +1,96 @@
-import { useEditStrategyCtx } from './EditStrategyContext';
 import { Link, useParams } from '@tanstack/react-router';
+import { ReactComponent as IconDisposable } from 'assets/icons/disposable.svg';
+import { ReactComponent as IconRecurring } from 'assets/icons/recurring.svg';
+import { ReactComponent as IconOverlapping } from 'assets/icons/overlapping.svg';
+import { Tooltip } from 'components/common/tooltip/Tooltip';
+import { useEditStrategyCtx } from 'components/strategies/edit/EditStrategyContext';
+import { EditDisposableStrategySearch } from 'pages/strategies/edit/prices/disposable';
+import { EditRecurringStrategySearch } from 'pages/strategies/edit/prices/recurring';
+import { EditOverlappingStrategySearch } from 'pages/strategies/edit/prices/overlapping';
+import { Strategy } from 'libs/queries';
 import {
   EditTypes,
   toDisposablePricesSearch,
   toOverlappingPricesSearch,
   toRecurringPricesSearch,
 } from 'libs/routing/routes/strategyEdit';
-import { ReactComponent as IconDisposable } from 'assets/icons/disposable.svg';
-import { ReactComponent as IconRecurring } from 'assets/icons/recurring.svg';
-import { ReactComponent as IconOverlapping } from 'assets/icons/overlapping.svg';
-import { FC } from 'react';
 
-interface Props {
-  editType: EditTypes;
-}
-export const EditPriceNav: FC<Props> = ({ editType }) => {
+const links = [
+  {
+    label: 'Limit / Range',
+    svg: <IconDisposable className="hidden size-14 md:inline" />,
+    to: '/strategies/edit/$strategyId/prices/disposable',
+    text: 'Buy or sell at a specific price, or gradually scale in or out of a position.',
+    id: 'disposable',
+  },
+  {
+    label: 'Recurring',
+    svg: <IconRecurring className="hidden size-14 md:inline" />,
+    to: '/strategies/edit/$strategyId/prices/recurring',
+    text: 'Create an automated trading cycle of buy low/sell high with two separate orders.',
+    id: 'recurring',
+  },
+  {
+    label: 'Concentrated',
+    svg: <IconOverlapping className="hidden size-14 md:inline" />,
+    to: '/strategies/edit/$strategyId/prices/overlapping',
+    text: 'Buy and sell within custom parameters with custom fee tier and auto-compounding fees.',
+    id: 'overlapping',
+  },
+] as const;
+
+export const toPricesSearch = (
+  strategy: Strategy,
+  editType: 'editPrices' | 'renew',
+  strategyType: 'disposable' | 'recurring' | 'overlapping'
+):
+  | EditDisposableStrategySearch
+  | EditRecurringStrategySearch
+  | EditOverlappingStrategySearch => {
+  switch (strategyType) {
+    case 'disposable':
+      return toDisposablePricesSearch(strategy, editType);
+    case 'recurring':
+      return toRecurringPricesSearch(strategy, editType);
+    case 'overlapping':
+      return toOverlappingPricesSearch(strategy, editType);
+  }
+};
```

Comment:
Usually switch can be replace with an object : 
```ts
const priceSearchFn = {
  disposable: toDisposablePricesSearch,
  recurring: toRecurringPricesSearch,
  overlapping: toOverlappingPricesSearch
}
```

Reply:
Done